### DISCUSSION TITLE
Any Recent Installs on Ubuntu?

### DISCUSSION THREAD

**Discussion by kiot389** [Security-Onion-Solutions/securityonion#6892] [General] [answered]
Has anyone recently installed on Ubuntu 18.04? I'm in an endless loop and can't imagine someone installing Ubuntu, clones git, runs so-setup and they're done. If you've recently deployed, did you have to do anything else specific, install any program at all? Did you run into any stopping points you had to resolve or prep?

**Comment by RobertClum** [comment]
I did the install on ubuntu 20.04 and it seemed to work well. I did have one issue with Wazuh agent communication, but I never resolved that issue before reinstalling with the SO ISO.

**Reply by kiot389** [reply]
@RobertClum I resolved that by installing curl. I was surprised its not part of the OS. Its one of the reason I posted...was wondering if there's other preps. I'll try 20.04.

**Reply by RobertClum** [reply]
Oh, good to know. I didn't have to install curl on 20.04 so it must already have curl installed.

**Comment by dougburks** [comment, marked as answer]
@kiot389 I've created the following issue for the curl dependency:
https://github.com/Security-Onion-Solutions/securityonion/issues/6899

In the meantime, I've updated the documentation to include curl as a dependency:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

I can confirm that, after installing `curl`, installation on Ubuntu 18.04 works for me:

![Ubuntu 18 04 - 1](https://user-images.githubusercontent.com/1659467/149939954-587e5be6-c5d1-40c3-ac0a-de0757e51501.png)

![Ubuntu 18 04 - 2](https://user-images.githubusercontent.com/1659467/149939957-35a38de3-8ec1-43c8-b61c-baf85d514a6b.png)

![Ubuntu 18 04 - 3](https://user-images.githubusercontent.com/1659467/149939959-9c829d44-0d9e-4abc-ba48-d4db40293a48.png)
